bloges56/DateSim
# Build Fixes

## Description
- Fixed Manager.cs not loading scripts correctly preventing the build version from working properly. The build version now works with a few bugs. 

## How to Test

1. In Unity, select `File` ->`Build Settings` 
2. Make sure all of the scenes are in the build settings with the SceneManager scene at the 0 index. <img width="1044" alt="image" src="https://user-images.githubusercontent.com/60618398/166496102-00bcdf12-73ed-425e-ac96-33ae65b416f2.png">
3. Select your Target Platform and settings.
5. If you want to show Errors and see the log, select `Development Build` and `Script Debugging` in the settings <img width="802" alt="image" src="https://user-images.githubusercontent.com/60618398/166496635-4e281e89-90d3-4ce6-8188-d9caa98f9192.png">
7. Click `Player Settings` on the bottom right of the Build Settings window to open up the Project Settings window.
8. Select `Script Execution Order` on the left side of the window.
9. Check that the scripts are in the following order: <img width="940" alt="image" src="https://user-images.githubusercontent.com/60618398/166497273-58e6bfd0-6f1e-4066-8c31-a0d5c16e5682.png">
11. If they're not, re-arrange them. If they're not present, select the `+` at the bottom right of the section and select those scripts to add them to the Execution Order. Then, re-arrange them into the correct order. 
12. If needed, click apply on the Script Execution Order settings. Then, close out of the project settings. 
13. Inside of the Build Settings window, select build and save the executable. 
14. Run the game and test previous 

## Checklist

- [ ] Does the game start without any errors?
- [ ] Are you able to play games? 

## Things to do

- [ ] Fix scoreboard to be binary
- [ ] Fix mouse texture issue in the Shooter game
- [ ] Fix sprite loading issue(out of order)
